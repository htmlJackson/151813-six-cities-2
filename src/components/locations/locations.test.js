import React from "react";
import renderer from "react-test-renderer";

import Locations from "./Locations.jsx";

it(`Locations correctly renders after relaunch`, () => {
  const tree = renderer.create(<Locations offers={[{}, {}, {}, {}]} citiesList={[`Bobruisk`, `Default city`, `Kishinev`]} city={`Bobruisk`} onCityClick={jest.fn()} />).toJSON();

  expect(tree).toMatchSnapshot();
});
