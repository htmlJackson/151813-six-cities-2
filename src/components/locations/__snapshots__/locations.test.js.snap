// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locations correctly renders after relaunch 1`] = `
<section
  className="locations container"
>
  <ul
    className="locations__list tabs__list"
  >
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item  tabs__item--active"
        href="#"
        onClick={[Function]}
      >
        <span>
          Bobruisk
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item "
        href="#"
        onClick={[Function]}
      >
        <span>
          Default city
        </span>
      </a>
    </li>
    <li
      className="locations__item"
    >
      <a
        className="locations__item-link tabs__item "
        href="#"
        onClick={[Function]}
      >
        <span>
          Kishinev
        </span>
      </a>
    </li>
  </ul>
</section>
`;
